dodo.py

```diff
@@ -232,7 +232,21 @@ def archive_project(root='', name='all'):
 
 def task_build_project():
     """Build project with given name, assumes you are in an environment with required dependencies"""
+
+    def move_thumbnails(name):
```

Comment:
Rebased incorrectly or did you really add this?

Reply:
I believe @jsignell added this...

Reply:
Yes this allows people to add thumbnails. As long as they check them into a thumbnails directory in the project.